proxy/test/utils/customer-variables/selectors.test.ts

```diff
@@ -235,201 +234,217 @@ describe('customer variables selectors', () => {
       Buffer.from(variablesRecordStr),
     ])('positive scenario for custom origin with buffer secret', async (buffer) => {
-      mockSecret.asBuffer(buffer)
+      mock
+        .on(GetSecretValueCommand, {
+          SecretId: secretName,
+        })
+        .resolves({
+          SecretBinary: buffer,
+        })
 
       await runAssertions()
     })
 
     test('positive scenario for custom origin with string secret', async () => {
-      mockSecret.asString(variablesRecordStr)
+      mock
+        .on(GetSecretValueCommand, {
+          SecretId: secretName,
+        })
+        .resolves({
+          SecretString: variablesRecordStr,
+        })
 
       await runAssertions()
     })
 
-    test('positive scenerio for s3 origin with string secret', async () => {
-      const req: CloudFrontRequest = {
-        clientIp: '1.1.1.1',
-        method: 'GET',
-        uri: 'fpjs/agent',
-        querystring: 'apiKey=ujKG34hUYKLJKJ1F&version=3&loaderVersion=3.6.2',
-        headers: {},
-        origin: {
-          s3: {
-            domainName: 'adewe.cloudfront.net',
-            path: '/',
-            region: 'us',
-            authMethod: 'none',
-            customHeaders: {
-              fpjs_secret_name: [
-                {
-                  key: 'fpjs_secret_name',
-                  value: 'my_secret',
-                },
-              ],
-              fpjs_secret_region: [
-                {
-                  key: 'fpjs_secret_region',
-                  value: 'eu-west-1',
-                },
-              ],
-            },
-          },
-        },
-      }
-
-      mockSecret.asString(variablesRecordStr)
-
-      await runAssertions(req)
-    })
+    // test('positive scenario for s3 origin with string secret', async () => {
+    //   const req: CloudFrontRequest = {
+    //     clientIp: '1.1.1.1',
+    //     method: 'GET',
+    //     uri: 'fpjs/agent',
+    //     querystring: 'apiKey=ujKG34hUYKLJKJ1F&version=3&loaderVersion=3.6.2',
+    //     headers: {},
+    //     origin: {
+    //       s3: {
+    //         domainName: 'adewe.cloudfront.net',
+    //         path: '/',
+    //         region: 'us',
+    //         authMethod: 'none',
+    //         customHeaders: {
+    //           fpjs_secret_name: [
+    //             {
+    //               key: 'fpjs_secret_name',
+    //               value: secretName,
+    //             },
+    //           ],
+    //           fpjs_secret_region: [
+    //             {
+    //               key: 'fpjs_secret_region',
+    //               value: secretRegion,
+    //             },
+    //           ],
+    //         },
+    //       },
+    //     },
+    //   }
+
+    //   mock.on(GetSecretValueCommand, {
+    //     SecretId: secretName,
+    //   }).resolves({
+    //     SecretString: variablesRecordStr
+    //   })
+
+    //   await runAssertions(req)
+    // })
   })
 
-  describe('from secrets manager and headers', () => {
```

Comment:
we should consider deleting these codes if they are not needed anymore. They are in comment right now